src/plugins/intel_gpu/src/plugin/transformations/transpose_fusion.cpp

```diff
@@ -30,7 +30,39 @@ namespace ov::intel_gpu {
 
```

Comment:
random spot) could you update the PR description about why this is a problem/why it is required? I guess this is about `There was mismatch between graph transformation and onednn support for transpose-fusion`. Am I correct?

Reply:
@isanghao You're right. Updated title and description of PR. Thanks!